### PR TITLE
Removed unused Automatic Title from Admin Templates: Hathor & System

### PR DESCRIPTION
**Pull request by pe7er**
Removed "Automatic Title" check & use from Admin Templates: Hathor & System",
as the Module "Advanced" option "Automatic Title" is not used anymore.
(see other https://github.com/joomla/joomla-cms/pull/5789 )

Please test Titles of Modules in Admin Cpanel view with Hathor & System admin templates.

### CODE REVIEW

File: administrator/templates/hathor/html/com_cpanel/cpanel/default.php [hunk 1]
```diff
@@ -36,20 +36,7 @@
...
-	}
-	elseif (method_exists('mod'.$module->name.'Helper', 'getTitle'))
-	{
-		echo JHtml::_('sliders.panel', call_user_func_array(array('mod' . $module->name . 'Helper', 'getTitle'), array($params)), 'cpanel-panel-' . $module->name);
-	}
-	else
-	{
-		echo JHtml::_('sliders.panel', JText::_('MOD_' . $module->name . '_TITLE'), 'cpanel-panel-' . $module->name);
-	}
+        echo JHtml::_('sliders.panel', $module->title, 'cpanel-panel-' . $module->name);
```
Comment (zero-24) [inline administrator/templates/hathor/html/com_cpanel/cpanel/default.php]: here is one spacve to much :smile:

File: administrator/templates/system/html/modules.php [hunk 2]
```diff
@@ -80,18 +80,7 @@ function modChrome_sliders($module, &$params, &$attribs)
...
-		}
-		elseif (method_exists('mod' . $module->name . 'Helper', 'getTitle'))
-		{
-			echo JHtml::_('sliders.panel', call_user_func_array(array('mod' . $module->name . 'Helper', 'getTitle'), array($params, $module)), 'module' . $module->id);
-		}
-		else
-		{
-			echo JHtml::_('sliders.panel', JText::_('MOD_' . $module->name . '_TITLE'), 'module' . $module->id);
-		}
+                echo JHtml::_('sliders.panel', $module->title, 'module' . $module->id);
```
Comment (zero-24) [inline administrator/templates/system/html/modules.php]: same here

File: administrator/templates/system/html/modules.php [hunk 3]
```diff
@@ -104,18 +93,7 @@ function modChrome_tabs($module, &$params, &$attribs)
...
-		}
-		elseif (method_exists('mod' . $module->name . 'Helper', 'getTitle'))
-		{
-			echo JHtml::_('tabs.panel', call_user_func_array(array('mod' . $module->name . 'Helper', 'getTitle'), array($params)), 'module' . $module->id);
-		}
-		else
-		{
-			echo JHtml::_('tabs.panel', JText::_('MOD_' . $module->name . '_TITLE'), 'module' . $module->id);
-		}
+                echo JHtml::_('tabs.panel', $module->title, 'module' . $module->id);
```
Comment (zero-24) [inline administrator/templates/system/html/modules.php]: and here :smile:

**Comment by pe7er** [comment]
oops.. thanks!
I will fix it.

**Comment by pe7er** [comment]
For some reason my Netbeans now adds spaces when I use tab....
I have corrected the patch again, and changed spaces for tabs.

File: administrator/templates/system/html/modules.php [hunk 4]
```diff
@@ -80,19 +80,8 @@ function modChrome_sliders($module, &$params, &$attribs)
...
-		{
-			echo JHtml::_('sliders.panel', call_user_func_array(array('mod' . $module->name . 'Helper', 'getTitle'), array($params, $module)), 'module' . $module->id);
-		}
-		else
-		{
-			echo JHtml::_('sliders.panel', JText::_('MOD_' . $module->name . '_TITLE'), 'module' . $module->id);
-		}
-		echo $content;
+		echo JHtml::_('sliders.panel', $module->title, 'module' . $module->id);
+	echo $content;
```
Comment (zero-24) [inline administrator/templates/system/html/modules.php]: sorry @pe7er can you add here the tab back?

Comment (pe7er) [inline administrator/templates/system/html/modules.php]: yes, thanks @zero-24 !

**Comment by brianteeman** [comment]
all looks ok to me<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="http://issues.joomla.org/tracker/joomla-cms/5792">issues.joomla.org/joomla-cms/5792</a>.</sub>

**Comment by waader** [comment]
@test success!

**Comment by brianteeman** [comment]
Has two tests - setting RTC - thanks<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="http://issues.joomla.org/tracker/joomla-cms/5792">issues.joomla.org/joomla-cms/5792</a>.</sub>

**Comment by wilsonge** [comment]
Fixed with https://github.com/joomla/joomla-cms/commit/a453f5ec88fff4895d0e8b402d868d33e2ee7b78

Thanks Peter!

**Pull request closed without merging**